math/src/field/element.rs

```diff
@@ -411,6 +411,14 @@ impl<F: IsPrimeField> FieldElement<F> {
     pub fn legendre_symbol(&self) -> LegendreSymbol {
         F::legendre_symbol(&self.value)
     }
+
+    /// Creates a `FieldElement` from a hexstring. It can contain `0x` or not.
+    /// Returns an `CreationError::InvalidHexString`if the value is not a hexstring
```

Comment:
I think it also returns the CreationError when the hex is too large to fit in a u64.